Android/toonDra/app/build.gradle

```diff
@@ -28,4 +28,5 @@ dependencies {
   compile fileTree(include: ['*.jar'], dir: 'libs')
   compile 'com.android.support:appcompat-v7:24.2.1'
   testCompile 'junit:junit:4.12'
+  compile 'com.android.support:design:24.2.1'
```

Comment:
I first grouped it along with others but Android Studio automatically replaced it like above.